tests/fixtures/debian/kafka-streams-examples/standalone-network.yml

```diff
@@ -0,0 +1,117 @@
+---
+version: '2'
+
+networks:
+  # The name of the network defined here is used in `tests/test_kafka_streams_examples.py`
+  # as part of the 'NetworkMode' host config for some tests when running docker commands.
+  acme:
+    driver: bridge
+
+services:
+
+  ###
+  ### Bridge mode
+  ###
+  ### In this file, we use the port convention for bridge mode:
+  ### - Host port      = 2<usual_port> (example: 29092 for a Kafka broker that runs at port 9092)
+  ### - Container port = 1<usual_port> (example: 19092 for a Kafka broker that runs at port 9092)
+
+  zookeeper-bridge:
+    image: confluentinc/cp-zookeeper:3.2.0
+    labels:
+    - io.confluent.docker.testing=true
+    networks:
+    - acme
+    ports:
+    - "22181:12181"
```

Comment:
Yeah, I think I understand how that works yet.  The reason why I used non-default ports is because it is (IMHO) much clearer which port "variant"/network you are testing against with `tests/*`.  I found the existing test code (e.g. for `kafka-rest`) confusing because it was hard to untangle which test for "localhost" and "<default port>" would go against the host network vs. the container network and so on.  Perhaps that is clear for Docker vets, but as a beginner I was confused initially.